packages/ui-components/src/components/alert/alert.config.ts

```diff
@@ -0,0 +1,9 @@
+import { EIconName } from '../icon/icon.types';
+import { EAlertType } from './alert.types';
+
+export const ALERT_ICON_NAMES: { [key in EAlertType]: EIconName } = {
+	[EAlertType.Info]: EIconName.Error,
```

Comment:
Is this correct?

Reply:
The icon naming is a bit confusing but yes, this is correct, the icon is the same in the case of an Error or Information alert, only thing that changes is the fill color :P